server/backend.py

```diff
@@ -17,16 +18,21 @@
 since = 0
 quicker = 6
 begun = time.time()
-startDBTime(begun)
+
+network:int = 0
+
+class NetworkIDsToName(Enum): # i mean it is horrible, but technically it allows somewhat easier addition of networks (if that ever happens, which i doubt)
```

Comment:
Could this instead be an [`IntEnum`](https://docs.python.org/3/library/enum.html#enum.IntEnum) or similar? This would allow lines such as the if/else statement to be written as
```python
if network == NetworkIDsToName.nintendo:
     # […]
elif network == NetworkIDsToName.pretendo:
     # […]
```

On a related note, enum members are written in all capitals (e.g. `NINTENDO`).

Reply:
I'll move it to a separate file, and change it to IntEnum (if my code lets me lol), but I can't quite change it to uppercase, as it's used for the database name, and some UI elements, so uppercase it would cause issues. If you look in the code, my casing is all over the place. I normally program in C# and JS, so my casing is coming from there, which isn't best practice for python.

Reply:
Changed it to a IntEnum, but I don't think I'll change it to uppercase.